prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/ND/nd-doh.yml

```diff
@@ -0,0 +1,109 @@
+# To submit this to localhost, run:
```

Comment:
general comment here that Platform has introduced features that mean we shouldn't store these in the receivers/STLTs/** folders, but move them back to the prime-router/settings/Prod or prime-router/settings/stage folders